src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/NodeFactory.cs

```diff
@@ -44,6 +44,11 @@ public abstract partial class NodeFactory
             ObjectDataInterner dataInterner)
         {
             _target = context.Target;
+
+            InitialInterfaceDispatchStub = _target.Architecture == TargetArchitecture.ARM
+                ? new InitialInterfaceDispatchStubNode()
```

Comment:
I am wondering whether the special ARM dispatch stub wrapper is still necessary.

Reply:
I can try to delete that in a different PR and see what the CI says, but I don't have hardware to test it on.

Reply:
Testing it in CI should be good enough if you would like to give it a try.

I think this was only needed for multi-.dll Windows arm32 support that we do not have anymore.